src/main/java/seedu/address/model/person/DisplayPhoto.java

```diff
@@ -16,14 +18,14 @@
  * Guarantees: immutability and validity.
  */
 public class DisplayPhoto {
-
-    public static final String MESSAGE_PHOTO_CONSTRAINTS = "Display photo: "
-            + "specified file does not exist or it exceeded maximum size of 1MB.";
-
-    /* Display photos for sample persons in SampleDataUtil */
+    // Display photos for sample persons in SampleDataUtil
     public static final String SAMPLE_PHOTO_ALEX = "/images/sample_dp_one.png";
```

Comment:
Actually, can we rename `SAMPLE_PHOTO_ALEX` to simply `SAMPLE_PHOTO`?

Reply:
Yeap we could, I initially wanted to add more photos for persons in `SampleDataUtil` e.g. `SAMPLE_PHOTO_BERNICE`

Reply:
Updated 😝